### ISSUE TITLE
Gaining More than 4 groups

### ISSUE THREAD

**Issue by crispd** [crispd/issues#11]
Food for thought. It was mentioned that because of how frequently some devices get gained together, it may be desirable to have them gained as default. So a screen would come up with the given devices gained together.
I could see a number of issues with this but it does bring up the question of what is possible for addressing the want for more than 4 groups? In almost every scenario where tuning screens are used for lline and lline extension tuning screens there are at least 6 standard groups. The devices in these groups are never tuned individually because in most cases it would ruin the optical set up.

**Comment by dxmaxwell** [comment]
Adding more (or less) gaining groups is easy.  I only picked 4 initially because I was thinking that each group would correspond to a knob on a knob box and the knob boxes we have now have 4 knobs.  Assigning a default group to a device sounds like a good idea to me, the only question is where we store that information.  The easiest way to implement this is to add a column to the configuration file.

**Comment by crispd** [comment]
This is the easiest way for now, but the more information we put in this file, the harder it becomes to both read, and adjust/create these configuration files with a later application.
In the case where we end up adding 15 unique values to the configuration file to test various solutions or improvements but only end up wanting/using 6 of them, how difficult is it to suppress or completely leave out those values in successive implementations of your screen?

**Comment by crispd** [comment]
I like the consideration of the knob solution.

**Comment by crispd** [comment]
Is there any precedent for using a small touch screen for knob control? Do touch-enabled widgets exist? I'm curious.

**Comment by dxmaxwell** [comment]
Yes, touch enabled GUI components are supported, but I've never used them.
